lib/internal/modules/esm/translators.js

```diff
@@ -64,6 +66,7 @@ const { emitWarningSync } = require('internal/process/warning');
 
 let cjsParse;
 async function initCJSParse() {
+  // eslint-disable-next-line node-core/prefer-primordials
   if (typeof WebAssembly === 'undefined') {
```

Comment:
Wouldn't it also work if we imported the WebAssembly object from primordials?

Reply:
IIRC `WebAssembly` is not available when compiling with `--jitless` (https://github.com/nodejs/node/pull/35583). Not sure it changes something for this check though.

Reply:
we can refer to the one from primordials here.